package.json

```diff
@@ -31,9 +31,11 @@
     "ember-uuid": "1.0.0"
   },
   "devDependencies": {
+    "bootstrap": "3.3.7",
     "broccoli-asset-rev": "^2.4.5",
     "broccoli-merge-trees": "^1.2.1",
     "broccoli-static-compiler": "^0.2.1",
+    "ember-bootstrap": "1.0.0-alpha.10",
```

Comment:
I'd rather not use this heavy addon since we don't use it anywhere else in MapGeo, and most likely we'll be updating to material-design or BS4 in the near future.